components/Modal/CreateImage.js

```diff
@@ -128,12 +129,16 @@ class CreateImage extends React.Component {
                   ><FormattedMessage defaultMessage="Blueprint" /></label>
                   <div className="col-sm-9">
                     <p className="form-control-static">{this.props.blueprint.name}</p>
+                    	<label
+                    		data-id="cmpsr-form-ctrl"
+                    	>
```

Comment:
The element we want to target is the paragraph `<p>` element that contains the blueprint name. Instead of adding a new `<label>` element here, the `data-id` attribute should be added to the existing `<p>` element. 

Also, `"cmpsr-form-ctrl"` is a little too generic for our purposes. Maybe `"cmpsr-form-blueprint"`?

So after these changes, it would be something like:
```
<p 
  className="form-control-static"
  data-id="cmpsr-form-blueprint"
>{this.props.blueprint.name}</p>
```

BTW, in looking at the existing `id` values being used in this form, they're not that great and they're all really generic. Sorry for having bad examples for you to work from. 😬

Reply:
This makes sense ! I'll fix this